software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -542,4 +602,10 @@ public String getOsMajorVersion() {
         String osVersion = osDetails.getVersion();
         return osVersion.contains(".") ? osVersion.substring(0, osVersion.indexOf(".")) : osVersion;
     }
+
+    private void addRiakOnPath(ScriptHelper scriptHelper) {
+        Map<String, String> newPathVariable = ImmutableMap.of("PATH", sbinPath);
+//        log.warn("riak command not found on PATH. Altering future commands' environment variables from {} to {}", getShellEnvironment(), newPathVariable);
```

Comment:
For distributions that require this, It is annoying to show it all the time in the log